.github/workflows/codespell.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Codespell
+
+on:
+  push:
+
+  pull_request:
+    branches:
+      - master
```

Comment:
```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

Reply:
Done, just removed some spaces to please [yamlllint](https://github.com/adrienverge/yamllint). But I can put them back, sometimes this linter is too intrusive :smile: